setup.py

```diff
@@ -11,7 +11,8 @@
       py_modules=['tap_square'],
       install_requires=[
           'singer-python==5.9.0',
-          'squareup==5.3.0.20200528'
+          'squareup==5.3.0.20200528',
+          'backoff==1.8.0',
```

Comment:
PyPI lists the most recent version as `1.10.0`, is there any reason we picked `1.8.0`?

Reply:
singer-python lists 1.8.0 - pip install reports a conflict if I try to use the latest. The alternative is a PR to update singer-python library and release a new one, which I'm happy to do since it would be a minor version bump at the the most and change nothing really about the library